include/irr/core/math/aabb2dSIMDIntBase.h

```diff
@@ -48,7 +49,7 @@ class aabb2dSIMDBase : public AlignedBase<_IRR_VECTOR_ALIGNMENT>
 	inline void reset(const VECTOR_TYPE& point)
 	{
 		CRTP<VECTOR_TYPE> pointBox(point);
-		internalBoxRepresentation = pointBox.internalBoxRepresentation;
+		*this = pointBox;
```

Comment:
nope, you need to downcast the pointer and call the CRTP version of operator=

Reply:
got it, just in case copy assigment is overrided